.babelrc

```diff
@@ -1,3 +1,15 @@
 {
-  "presets": ["react", "es2015", "stage-1"]
+  "presets": ["react", "es2015", "stage-1"],
+  "env": {
+       "server": {
+           "plugins": [
+               [
+                   "css-modules-transform", {
+                       "generateScopedName": "[hash:8]",
+                       "extensions": [".css"]
```

Comment:
~we've got some linter differences between this and https://github.com/LearnersGuild/talent/pull/70~ edited: not in this file (was thinking of trailing commas)